spec/features/task_list_spec.rb

```diff
@@ -5,26 +5,76 @@
 RSpec.describe "Task List", type: :feature do
   before do
     login_as :engineer_user
-
-    create_list(:task, 3, completed_at: nil)
-    create_list(:task, 3, completed_at: 3.days.ago)
   end
 
-  it "shows all the tasks" do
-    visit tasks_path
+  context "when visiting index of tasks without board" do
+    before do
+      create_list(:task, 2, completed_at: nil)
+      create_list(:task, 3, completed_at: 3.days.ago)
+      create(:task, title: "Boardless Task")
```

Comment:
Probably overkill to hardcode the name here to be tested later if you had a let that was the "boardless list" then you can just test `boardless_list.first.name` or similar